### PR TITLE
User method

### LINKED ISSUE #83: [sign-up] #83 Adding a user in the DB when user signed up

### PR DESCRIPTION
**Pull request by morbitton**
fix #83 

@lioramilbaum 
Barak and I work together @BarakMerdler

### CODE REVIEW

**Comment by lmilbaum** [comment]
@BarakMerdler please add a comment

File: app.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```
Comment (lmilbaum) [inline app.py]: why?

Comment (morbitton) [reply app.py]: my mistake.. didn't notice i deleted "3" from the line

**Comment by BarakMerdler** [comment]
comment

**Comment by morbitton** [comment]
@lioramilbaum we close it, because @BarakMerdler wrote this code at #176 PR, and added about it.

**Pull request closed without merging**